plugin/pkg/admission/serviceaccount/admission.go

```diff
@@ -57,6 +57,8 @@ type serviceAccount struct {
 
 	// LimitSecretReferences rejects pods that reference secrets their service accounts do not reference
 	LimitSecretReferences bool
+	// RequireAPIToken determines whether pod creation attempts are rejected if no API token exists for the pod's service account
+	RequireAPIToken bool
```

Comment:
If it's for testing, can we lowercase it? Or remove entirely works for me too. :)